firedrake/utility_meshes.py

```diff
@@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, L):
```

Comment:
That suggests intercepting the PeriodicRectangleMesh construction, rather than making a new class?